### PR TITLE
nextjs cleanup

### PR DESCRIPTION
**Pull request by joshua-isaac**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,3 +1 @@
-module.exports = {
```
Comment (joelvarty) [inline next.config.js:1]: You actually need to leave this in there for it to work on netlify - can you test that?

File: pages/api/preview.js [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
-	//enable preview mode
-	res.setPreviewData({})
-
-	// Redirect to the slug
-	res.writeHead(307, { Location: previewUrl })
-	res.end()
-
-}
\ No newline at end of file
+  //validate our preview key, also validate the requested page to preview exists
```
Comment (joelvarty) [inline pages/api/preview.js]: Only thing I;m not sure of here is whether this will work on netlify - can you test?

**Review by joelvarty — CHANGES_REQUESTED** [review]

File: next.config.js (hunk 1 above)
Comment (joshua-isaac) [reply next.config.js:1]: @joelvarty it appears like it's still able to deploy to netlify without `target: "serverless"`.

see here: https://relaxed-panini-96053e.netlify.app/ - preview mode is still weird though

File: pages/api/preview.js (hunk 2 above)
Comment (joshua-isaac) [reply pages/api/preview.js]: @joelvarty i actually didn't change anything here, just formatting!

**Pull request merged**